modules/databases/download_references.nf

```diff
@@ -7,6 +7,6 @@ process download_references {
     script:
         """
         git clone https://github.com/mult1fractal/WtP_phage_reference_db.git
-        cat WtP_phage_reference_db/*.fa > phage_references.fa
+        cat WtP_phage_reference_db/fasta-files/*.fa > phage_references.fa
```

Comment:
this is the git where I put all the 4000 phage fasta files in the top layer of the git repo

cleaned up now --> fasta-files is the folder where I put all the files